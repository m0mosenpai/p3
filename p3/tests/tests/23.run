../solution/wsh <tests/23.wsh
