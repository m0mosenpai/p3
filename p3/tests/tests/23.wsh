local a=lorem_ipsum_dolor_sit_amet
local a=hello
echo $a
