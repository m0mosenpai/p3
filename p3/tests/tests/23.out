wsh> wsh> wsh> hello
wsh> 